packages/inertia-react/index.d.ts

```diff
@@ -30,9 +30,17 @@ interface InertiaLinkProps extends React.HTMLAttributes<HTMLAnchorElement> {
   preserveState?: boolean | ((props: Inertia.PageProps) => boolean)
   replace?: boolean
   only?: string[]
+  onCancelToken: (cancelToken: string) => void
```

Comment:
Are you sure cancelToken is a string? 
If memory serves me correctly, it's an object on which you can call `.cancel()`

Reply:
Good catch! Had it confused with something else. I think it's a `CancelTokenSource` object.